src/mpyl/schema/mpyl_config.schema.yml

```diff
@@ -292,6 +292,8 @@ definitions:
         type: string
       password:
         type: string
+      provider:
+        type: string
```

Comment:
You should probably add an enum for both the schema and the strings used in mpyl itself. See `deployAction` and how the values are used in mpyl as an example

Reply:
Is this necessary? I'm not sure how long we want to have both registries. The goal is to shut down azure asap

Reply:
Well it can't hurt either way, at least people won't be able to enter a random string that we haven't implemented.